src/main/java/org/geppetto/frontend/controllers/WebsocketConnection.java

```diff
@@ -185,6 +188,20 @@ protected void onTextMessage(CharBuffer message) throws IOException
 				connectionHandler.getVersionNumber(requestID);
 				break;
 			}
+			case USER_PRIVILEGES:
+			{
+				connectionHandler.checkUserPriviledges(requestID);
+				break;
+			}
+			case PROJECT_PERSISTENCE_STATE:
```

Comment:
Yes, it's been used already, but it is not being updated when the project loads and this extra check was added. Will attempt to set the persisted flag when the project loads.